octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+
+        int c = a + b;
+
```

Comment:
[Severity: Minor] The output message is hardcoded and does not provide flexibility for different operations.
Consider using a method to handle output formatting, which can be reused for different operations (e.g., `public static void printResult(String operation, int result)`).
Rationale: This promotes code reuse and makes it easier to extend the application to support more operations in the future.
Reference: DRY (Don't Repeat Yourself) principle.